src/actions/arcActions.ts

```diff
@@ -619,6 +619,7 @@ export function createProposal(daoAvatarAddress: string, title: string, descript
       dispatch(push("/dao/" + daoAvatarAddress));
 
       await contributionRewardInstance.proposeContributionReward({
+        title,
```

Comment:
huh? why is this here? isn't this calling arc.js, why would you pass in a title?

Reply:
weird, ok